src/gfx/VolumeMesh.js

```diff
@@ -80,35 +78,38 @@ VolumeMesh.prototype._updateVertices = (function() {
     [3, 7, 1, 1, 0]
   ];
 
-  var edgeIntersections = [];
-  for (var j = 0; j < 12; ++j) {
-    edgeIntersections.push(new THREE.Vector3());
-  }
-
+  static _edgeIntersections = [];
```

Comment:
In the old code version this array was initialized only once in order to improve performance.